File: src/main/java/com/will/utils/CollectionUtils.java
package com.will.utils;

import java.util.Collection;

/**
 * Created by Will on 2016/9/5 13:54.
 */
public class CollectionUtils {
    public static boolean isEmpty(Collection<?> distances) {
        return distances == null || distances.size() == 0;
    }
}

File: src/main/java/com/will/utils/CollectionUtils.kt
package com.will.utils

/**
 * Created by zoumy on 2017/5/12 12:39.
 */
object CollectionUtils {
    fun isEmpty(distances: Collection<*>?): Boolean {
        return distances == null || distances.isEmpty()
    }
    fun  isNoteEmpty(distances: Collection<*>?):Boolean{
        return !isEmpty(distances)
    }
}